### ISSUE TITLE
Large File Tests failing on ARM

### ISSUE THREAD

**Issue by WardF** [Unidata/netcdf-c#629]
Possibly related to the recent spate of merges, so that will be put on hold until this is tracked down.  The tests are running but so far the following tests are failing on ARM:

* nc_test/tst_big_var2
* nc_test/tst_big_rvar
* nc_test/tst_big_var

I will update this list when the tests finish (they take around an hour on ARM) and post a link to the cdash entry with the failure information.  Once we have that I can do a git bisect to track down the failure.

* Update - LFS tests on OSX are working (64-bit), will see if 32-bit work as well.
* Update - LFS tests work on 64-bit windows.  Will try 32-bit next.

**Comment by WardF** [comment]
It is pure supposition that it's related to the recent spate of merges; that was simply my first thought since I haven't been doing LFS tests each time.

**Comment by edhartnett** [comment]
As for me, I just noticed large file tests and added them to some CI builds today.

I am still hunting for my rasberry pi so I can set up some ARM builds myself...

**Comment by WardF** [comment]
Fortunately, the issue appears to be environmental. Fixed.

**Comment by WardF** [comment]
I will test this branch shortly, thank you very much @wkliao

**Comment by edhartnett** [comment]
@WardF let me know the results. If it fixes the issue I will include this PR in my mega-merge PR. (Or you can just go ahead and merge directly to master.)

**Comment by WardF** [comment]
Running now; just FYI, between a combination of slow processor and the terrible I/O throughput of the flash drive, these take a fairly long time to run on ARM. But, they are running :)

**Comment by edhartnett** [comment]
Yes, I have a matrix build on mine that takes forever to build a bunch of combinations. But I will run it on my current branch and go off and have some beer and pizza.
